etc/env/example.env

```diff
@@ -274,11 +272,6 @@ OPA_URL=http://opa:${OPA_PORT}
 
 CANDIG_AUTHZ_SERVICE_PORT=8182
 
-# same as former candig-server port - used to relay to new c-s port
```

Comment:
just want to confirm that we are deprecating arbiter code

Reply:
yes, we are removing arbiter.  

arbiter was never and is of no use as proposed.  arbiter's purpose was proposed to be the policy enforcer but that does not work unless arbiter sits in the middle of the application and the database and at the same time speaks the said database's wire protocol to munge sql request to the database.  which is not something that we are doing or will do (although this is a good research problem, right @ljdursi ? ).

Reply:
this reminds me that i forgot to remove arbiter from the auth module list.  thanks for mentioning.